src/rpc/payment.rs

```diff
@@ -118,18 +119,23 @@ pub(crate) struct SendPaymentCommandParams {
 pub struct HopHint {
     /// The public key of the node
     pub pubkey: Pubkey,
-    /// The funding transaction hash of the channel outpoint
-    pub channel_funding_tx: Hash256,
-    /// inbound or outbound to use this channel
-    pub inbound: bool,
+    /// The outpoint of the channel
+    #[serde_as(as = "EntityHex")]
+    pub channel_outpoint: OutPoint,
+
```

Comment:
any special reason to change from tx hash to channel outpoint?

Reply:
Only to be more consistent with other source code in the code base. Either tx hash and outpoint is fine for me. We just need to be consistent.